### ISSUE TITLE
OtlpHttpTransportFactory requires protobuf implementation, but not mentioned in docs

### ISSUE THREAD

**Issue by tigrannajaryan** [open-telemetry/opentelemetry-php#1065]
**Describe your environment** 
$ php -v
PHP 8.2.7 (cli) (built: Jun  8 2023 20:05:45) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.7, Copyright (c), by Zend Technologies

**Steps to reproduce**
I am following the instruction exporter example here https://opentelemetry.io/docs/instrumentation/php/exporters/

Executed `composer require  open-telemetry/exporter-otlp  php-http/guzzle7-adapter`

Using the following code:
```php
<?php

declare(strict_types=1);

use OpenTelemetry\API\Logs\LogRecord;
use OpenTelemetry\SDK\Sdk;
use OpenTelemetry\SDK\Common\Attribute\Attributes;
use OpenTelemetry\SDK\Common\Instrumentation\InstrumentationScopeFactory;
use OpenTelemetry\SDK\Common\Time\ClockFactory;
use OpenTelemetry\SDK\Logs\Exporter\ConsoleExporterFactory;
use OpenTelemetry\SDK\Logs\LoggerProvider;
use OpenTelemetry\SDK\Logs\Processor\SimpleLogsProcessor;
use OpenTelemetry\SDK\Logs\Processor\BatchLogsProcessor;
use OpenTelemetry\API\Common\Instrumentation\Globals;
use OpenTelemetry\Contrib\Otlp\OtlpHttpTransportFactory;
use OpenTelemetry\Contrib\Otlp\LogsExporter;

require 'vendor/autoload.php';


$transport = (new OtlpHttpTransportFactory())->create('http://jaeger:4318/v1/traces', 'application/json');

$exporter = new LogsExporter($transport);

$loggerProvider = new LoggerProvider(
    new SimpleLogsProcessor(
        (new ConsoleExporterFactory())->create()
    ),
    new InstrumentationScopeFactory(Attributes::factory())
);

Sdk::builder()
    ->setLoggerProvider($loggerProvider)
    ->setAutoShutdown(true)
    ->buildAndRegisterGlobal();


$logger = Globals::loggerProvider()->getLogger('demo', '1.0', 'http://schema.url', ['scopefoo' => 'scopebar']);

$record = (new LogRecord(['foo' => 'bar', 'baz' => 'bat', 'msg' => 'hello world']))
    ->setSeverityText('INFO')
    ->setSeverityNumber(9);

$logger->emit($record);
```

**What is the expected behavior?**
Logs exported via OTLP

**What is the actual behavior?**
Error:
```
Uncaught RuntimeException: No protobuf implementation found (ext-protobuf or google/protobuf) in /Users/tnajaryan/work/experiments/otel-php/vendor/open-telemetry/exporter-otlp/LogsExporter.php:33
Stack trace:
#0 /Users/tnajaryan/work/experiments/otel-php/otlpexport.php(23): OpenTelemetry\Contrib\Otlp\LogsExporter->__construct(Object(OpenTelemetry\SDK\Common\Export\Http\PsrTransport))
#1 {main}
  thrown in /Users/tnajaryan/work/experiments/otel-php/vendor/open-telemetry/exporter-otlp/LogsExporter.php on line 33
```

**Additional context**
Part of review https://github.com/open-telemetry/community/issues/1537

**Comment by brettmc** [comment]
related issue: #1061

**Comment by brettmc** [comment]
Fixed by #1067 and released in https://github.com/opentelemetry-php/gen-otlp-protobuf/releases/tag/1.0.0beta6
